doc/sphinx/source/recipes/recipe_python.rst

```diff
@@ -0,0 +1,67 @@
+.. _recipe_python:
+
+Example recipe Python
+=====================
+
+Overview
+--------
+
+This is an example recipe calling a simple diagnostic script written in Python.
+The recipe produces time series plots of global mean temperature and for the
+temperature in Amsterdam. It also produces a map of global temperature in
+January 2020.
+
+For detailed instructions on obtaining input data, please refer to
+:ref:`inputdata`. However, in case you just quickly want to run through the
+example, you can use the following links to obtain the data from ESGF:
+
+  * `BCC-ESM1 <http://cmip.bcc.cma.cn/thredds/fileServer/cmip6_data/CMIP/BCC/BCC-ESM1/historical/r1i1p1f1/Amon/tas/gn/v20181214/tas_Amon_BCC-ESM1_historical_r1i1p1f1_gn_185001-201412.nc>`_
```

Comment:
Downloading the data from China is very slow for me, I get a download speed of about 15KB/s. Maybe you could use a DKRZ host instead?

Reply:
Changed it myself